docs/spark-queries.md

```diff
@@ -394,3 +394,14 @@ spark.read.format("iceberg").load("db.table.files").show(truncate = false)
 // Hadoop path table
 spark.read.format("iceberg").load("hdfs://nn:8020/path/to/table#files").show(truncate = false)
 ```
+
+You can also inspect Iceberg metadata tables with the time travel feature:
```

Comment:
1. I think we tend to avoid second person 'you' in the docs, and just state that something can be done.

2. Also what do you think about making a separate section (Time Travel with Metadata Tables) and showing both the DF and the SQL way ?  Otherwise people may assume its only possible via dataframe.

3.  Also this is unrelated but I realized that this section (Inspecting with DataFrames) has the wrong heading level, it should be under the level of (Inspecting Tables), as that level is about all metadata tables.  Can you help fix it as well?